benchmarks/README.md

```diff
@@ -1,9 +1,16 @@
 # Relay Benchmarks
 
+## Requirements
+
+- PHP 7.4+
+- Redis 6.0+
+
 ## Installation
 
 First, follow the [installation instructions](https://relay.so/docs/installation) to set up the Relay extension for PHP.
 
+Make sure you have [installed PhpRedis](https://github.com/phpredis/phpredis/blob/develop/INSTALL.md) and, of course, [Redis itself](https://redis.io/docs/getting-started/installation/). When installing PhpRedis, say *yes* to all prompts about `enable igbinary serializer support` etc.
```

Comment:
I think the script should check dependencies before running. Is Relay installed? Is PhpRedis installed? Can Predis be loaded via Composer?